src/Molecules/ToastMessage/ToastMessage.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useImperativeHandle, useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+
+import { Flexbox } from '../..';
+import { BoxContainer, StyledTypography } from './ToastMessage.styled';
+import { Props } from './ToastMessage.types';
+
+const ToastMessages: React.FC<Props> = React.forwardRef(
+  ({ label, icon, linkButton, isVisible, timeout = 6000 }, ref) => {
```

Comment:
Since `isVisible` is optional in your types, you can default it to `false` to make sure you're always dealing with a boolean

```suggestion
  ({ label, icon, linkButton, isVisible = false, timeout = 6000 }, ref) => {
```